scripts/rollup/bundles.js

```diff
@@ -437,6 +437,7 @@ const bundles = [
       'ReactFlightNativeRelayServerIntegration',
       'JSResourceReferenceImpl',
       'ReactNativeInternalFeatureFlags',
+      'util',
```

Comment:
@sebmarkbage you commented that this should not be necessary. In practice the build fails without this so I assume `ReactServerStreamConfigNode.js` is getting pulled in by accident or something changed and now that is the expected file

Reply:
https://github.com/facebook/react/pull/24322/

Reply:
The React sync for React Native is currently failing because "util could not be found", I confirmed #24322 fixes it, how can we get it landed?